crates/marine-test-macro-impl/src/marine_test/config_utils.rs

```diff
@@ -34,12 +35,58 @@ impl<'m> Module<'m> {
     }
 }
 
+pub(crate) struct ConfigWrapper {
+    pub config: TomlAppServiceConfig,
+    pub resolved_modules_dir: PathBuf,
+}
+
+pub(crate) struct ProcessedService {
```

Comment:
As I understand you use `ProcessedService` only inside `service_generator.rs`, so I suggest to move it there.